src/firefly/java/edu/caltech/ipac/firefly/server/query/lsst/LSSTImageSearch.java

```diff
@@ -22,9 +20,9 @@
  * Created by zhang on 11/3/16.
  */
 public class LSSTImageSearch extends URLFileInfoProcessor {
-//    private static final Logger.LoggerImpl _log = Logger.getLogger();
+    private static final Logger.LoggerImpl _log = Logger.getLogger();
     private static String DAX_URL="http://lsst-qserv-dax01.ncsa.illinois.edu:5000/image/v0/";
-
+    private static FileInfo fileInfo;
```

Comment:
I did not clean up the codes after making changes.  Many variables were used before and were no longer needed after I made the code changes.  I will clean the codes and them merge.
Thanks!